fleetctl/destroy.go

```diff
@@ -35,6 +35,13 @@ Destroyed units are impossible to start unless re-submitted.`,
 }
 
 func runDestroyUnits(args []string) (exit int) {
+	// Even if no unit is given by user, it should not exit with error,
```

Comment:
I don't think the comment is necessary. Please just remove it here and everywhere else.